### ISSUE TITLE
Remove "Carbon" from w3 site page title

### ISSUE THREAD

**Issue by designertyler** [carbon-design-system/carbon-website#248]
The current page title is "Carbon Design System". As part of the Carbon scrub the new page title should be "Product Design" 
![Screen Shot 2018-10-10 at 10.47.25 AM.png](https://images.zenhubusercontent.com/583dd918b9dc3c622022afbf/1d2f8a37-22e6-4ca3-9482-e3a5aea6956a)

**Comment by designertyler** [comment]
@alisonjoseph heads up since we don't have a TT epic or tag

**Comment by alisonjoseph** [comment]
The title is being pulled from the config file, so not sure if this is something easily swapped for just the internal site but will take a look.

**Comment by kuehndaniel** [comment]
We need to scrub the whole site. Assigning to the volunteer - Chris Connors!

**Comment by alisonjoseph** [comment]
I don't understand how we can scrub the entire site when our packages are called `carbon-components` etc and our url is also www.carbondesignsystem.com, the internal and external websites are just one codebase with a few minor differences. This might be a bigger discussion on how we want to handle the name change issues.

**Comment by alisonjoseph** [comment]
I did a search for "carbon" in the website repo and found 3,608 results.

**Comment by jeoffw** [comment]
We don’t need to scrub the whole site right now. Let’s remove (if possible, otherwise deemphasize) carbon name references on the internal version of the home page, including nav.

We can address the rest later.

**Comment by chrisconnors-ibm** [comment]
@alisonjoseph this was always about what's surfaced in w3 - package names, etc. is a different problem.

**Comment by alisonjoseph** [comment]
Those links are links to public packages that's why I didn't update the names. The rest of the updates are live internally. (might have to clear cache to see)

**Comment by chrisconnors-ibm** [comment]
@designertyler @jeoffw shouldn't these two items be consistent:
![image](https://user-images.githubusercontent.com/35537391/46816131-4abd6400-cd4a-11e8-8d55-245ba3628790.png)

**Comment by jeoffw** [comment]
No **System** on the end. As for whether IBM needs to be in the hero space, I'll leave that to Tyler based on his discussions with Digital et al.

**Comment by jeoffw** [comment]
Could the tiles just be named:
- Design Kit
- Components
- Components React
- Components Angular
- CodePen
- Theming Sandbox

**Comment by alisonjoseph** [comment]
Yes, I can update them @jeoffw

**Comment by designertyler** [comment]
Yes to taking out "system" and updating the tiles without "Carbon"

[Digital has "IBM" in the header and does not use IBM in this section ](https://w3.ibm.com/design/digital/). They rely on the IBM logo in the top left.

There's no established naming convention (yet). My opinion is that we should use the full "IBM Product Design" when talking about our system to reinforce that it is aligned with the rest of IBM Design and it's the official way to build product at IBM.

We're also inconsistent across the board on how we're using emphasis/bold on "IBM" and "Design". We can shoot for consistency on our site for now

**TL;DR**
Always use "IBM Product Design" and put emphasis (semi-bold) on "IBM".
